x11-plugins/wmwork/wmwork-0.2.5.ebuild

```diff
@@ -4,8 +4,8 @@
 EAPI=0
```

Comment:
can you keep tabs on this, and call for stablereq in 30 days so we can clean out the old EAPI 0 ebuilds?

Reply:
@SoapGentoo 
Yes, thats actually the plan, but i wanted to push all remaining x11-plugins/wm* PRs first and create a big stable request bug for all x11-plugins/wm*  packages (which are at EAPI<=5)
Maybe you can have a look at my other PR's too ;-) They are almost all about upgrading packages to the latest EAPI.

Reply:
Yes, at this rate, a stabling round for all dockapps will be possible soon! Thanks again for these PR's @mm1ke sorry I lag a bit on the review side

Reply:
@voyageur 
sure no problem, i'm glad if i can help